test_data/yam/agama_sle_extended.yaml

```diff
@@ -11,3 +11,6 @@ repos:
     uri: https://download.opensuse.org/repositories/science/16.0/
     enabled: 'Yes'
     filter: name
+bootloader_timeout:
+  - interactive: 8
+    unattended: 15
```

Comment:
not like this, unfortunately we need two files.

Reply:
updated.